client/components/registration/registration.js

```diff
@@ -259,6 +259,7 @@ export default class Registration extends React.Component {
   };
 
   getLabelText = (plan) => {
+    /* disable ttag */
```

Comment:
nice :+1: 
is this to disable the warning?

Reply:
Yes